TODOS.md

```diff
@@ -0,0 +1,9 @@
+> @ generate:todos /Users/tdeekens/Development/nodejs
```

Comment:
Just renamed the file. It contains multiple todos so why not name it like that :)

Reply:
Wait isn’t this the problem I had too? Now we are checking in not so useful data in line #1 and #2

Reply:
‘	+> @ generate:todos /Users/tdeekens/Development/nodejs
’